packages/kit/src/vite/index.js

```diff
@@ -14,7 +14,12 @@ import { generate_manifest } from '../core/generate_manifest/index.js';
 import { get_runtime_directory, logger } from '../core/utils.js';
 import { find_deps, get_default_config as get_default_build_config } from './build/utils.js';
 import { preview } from './preview/index.js';
-import { get_aliases, resolve_entry } from './utils.js';
+import {
+	get_aliases,
+	resolve_entry,
+	format_illegal_import_chain,
```

Comment:
this doesn't look used. is there some way we can get typescript to report on unused imports or is this something we're missing from removing eslint? I think there's some stuff like this that's exported from `vite/utils.js` that might not need to be

Reply:
That function can likely be internal now since both functions that utilize it are in the same file.

Reply:
yep, this is an eslint casualty. fixed

Reply:
PR to stop it from happening again: https://github.com/sveltejs/kit/pull/5716